pages/design/atomic-template-empty.md

```diff
@@ -1,35 +1,50 @@
 ---
+id: 
 name:
 rule_type: atomic
 
 description: |
   This rule checks ...
 
-success_criterion:
-  - x.x.x # (Name of Success Criterion)
-
-test_aspects:
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:x.x.x: # <Name of Success Criterion> (A | AA | AAA)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied | further testing needed
+    inapplicable: satisfied | further testing needed
+    
+  aria11:some-id: # <Some heading in WAI-ARIA>
```

Comment:
For WAI-ARIA, can we use the id's that are actually used in WAI-ARIA 1.1, e.g. "#region" (https://www.w3.org/TR/wai-aria-1.1/#region) or "#requiredState" (https://www.w3.org/TR/wai-aria-1.1/#requiredState) ?

Reply:
Given we are using `x.x.x` everywhere, I think this should use the came convention here. 

It may be worth adding an example for each of the sections, but again this could be explained in this document - https://github.com/act-rules/act-rules.github.io/pull/488/files#r284189403